### PR TITLE
codeowners: update bh1749 owner

### PR DESCRIPTION
**Pull request by wlgrd**
Updates code owner of BH1749 library.
    
Signed-off-by: Christian Wilgaard <christian.wilgaard@nordicsemi.no>

### CODE REVIEW

**Comment by wlgrd** [comment]
CLA is signed, but for some reason not updated correctly. Assuming it just takes some time.

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
@jfischer-no ping

File: CODEOWNERS [hunk 1]
```diff
@@ -89,7 +89,7 @@ Kconfig*                                  @tejlmand
 /modules/                                 @tejlmand
 /modules/mcuboot/                         @nvlsianpu
 /modules/cjson/                           @simensrostad @plskeggs @sigvartmh
-/samples/sensor/bh1749/                   @wlgrd
+/samples/sensor/bh1749/                   @jfischer-no
```
Comment (jfischer-no) [inline CODEOWNERS:92]: What is the reason for this change?

Comment (wlgrd) [reply CODEOWNERS:92]: Since I am not an active contributor to nrfconnect, @carlescufi suggested to put you as code owner for this library.
